SEP-0009.md

```diff
@@ -0,0 +1,100 @@
+# SEP-0009 -- Release Pattern for SunPy Core
+
+| SEP           | 9                              |
+|---------------|--------------------------------|
+| title         | Release Pattern for SunPy Core |
+| author(s)     | Stuart Mumford, Russell Hewett |
+| contact email | stuart@cadair.com              |
+| date-creation | 2018-12-13                     |
+| type          | process                        |
+| status        | discussion                     |
+| discussion    |                                |
+
+# Introduction
+
+This SEP defines a release pattern for the SunPy core package and
+defines a versioning system. Since SunPy is being used more widely both by
+users as well as developers as a dependency, it is important to have a clear
+and predictable releases.  This is especially important to support other
+Python packages which have sunpy as a dependency (e.g. affiliated packages).
+A predictable release and support cycle for the SunPy core package will aid
+these packages in planning their own development cycle.
+
+The focus of this SEP is not to prescribe how quickly the core package improves
+through the introduction of new features or API changes but how and when
+release occur and how they are described.
+
+# Detailed Description
+
+## Release Schedule
+
+This SEP prescribes that there will be at least two releases of the sunpy core
+package per year with at least 4 months (preferably 6 months) between these
+releases. It is suggested that these occur in the months of May and
+November based on the current release schedule of the astropy core package. It
+is the responsibility of the lead developer to create a release schedule that
+is consistent with these requirement and to present it to the Board at the
+beginning of every year.  Releases are recommended to be timed to pertinent
+release cycles of upstream packages.
```

Comment:
And that would be published on the wiki? on the docs? as a new project/PR?
I would prefer we say here where to expect that information, and how long the beginning of the year would take... is that the first week of January? is that before end of February?

Reply:
Well I would imagine it in reality wont change very often, so I don't think timing is that important. To be honest I think the best canonical reference for the release schedule is this SEP?